Legacy docs review 3960 - metrics plugin (#8407)
Particular/docs.particular.net: monitoring/metrics/install-plugin_interval_metricsservicecontrol_[1,).partial.md

Specifies the maximum delay between sending metrics report messages.

> The metrics plugin buffers measurements and sends a metric report message as soon as the buffer is half full, without waiting for the interval to elapse. When an endpoint instance is idle there are no measurements to report, so no metric report message is sent. The plugin still sends a report at this interval regardless of load, which is how ServiceControl monitoring knows the instance is running rather than gone. When the endpoint is under load the interval between metric messages will be much shorter as this buffer fills faster.

Reviewer: "so no metric report message is sent" directly followed by "The plugin still sends a report at this interval regardless of load" seems contradictory.

Reviewer: ```suggestion
The metrics plugin buffers measurements and sends a metric report message as soon as the buffer is half full, without waiting for the interval to elapse. When an endpoint instance is idle the plugin will still send a report at this interval so that ServiceControl monitoring knows that the instance is still running. When the endpoint is under load, the time between metric messages will be much shorter as the buffer fills faster.
```

Example:

> If the system has 500 instances and the interval is set to 5 seconds, an idle system will send on average 6,000 messages per minute (500 × 12), which is 100 per second.

Reviewer: ```suggestion
If the system has 500 instances and the interval is set to 5 seconds, an idle system will send on average 6000 messages per minute (500 × 12), which is 100 per second.
```
Separator is specific to locale, maybe best to remove altogether?
